go/trace_ctx.go

```diff
@@ -33,6 +33,11 @@ func newTraceCtx(ctx context.Context, eventsChan chan TraceEvent, r wazero.Runti
 		return nil, err
 	}
 
+	if opts.ChannelBufferSize == 0 {
+		opts.ChannelBufferSize = 64 // set a reasonable minimum here so unset option doesn't block execution on an unbuffered channel
```

Comment:
Just for my clarity, this is a message size and not a byte size right? so 64 messages in the buffer?

Reply:
yes, 64 items on the channel until it blocks

Reply:
in Go, you call these "buffered channels" which have more than 0 capacity. or "unbuffered channels" which expect the channel to be read immediately unless a write will block. hence the name `ChannelBufferSize`, which can definitely be confusing if thinking about a more common byte buffer